test/libdnf/rpm/test_package.cpp

```diff
@@ -0,0 +1,346 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "test_package.hpp"
+
+#include "../utils.hpp"
+
+#include "libdnf/rpm/solv_query.hpp"
+
+#include <vector>
+
+
+CPPUNIT_TEST_SUITE_REGISTRATION(RpmPackageTest);
+
+
+libdnf::rpm::Package RpmPackageTest::get_pkg(const std::string &nevra) {
+    libdnf::rpm::SolvQuery query(sack.get());
+    query.ifilter_nevra(libdnf::sack::QueryCmp::EQ, {nevra});
+    CPPUNIT_ASSERT_MESSAGE("get_pkg(\"" + nevra + "\"): no package found.", query.size() > 0);
```

Comment:
What about to use assert `query.size() == 1`?

Reply:
Yeah.. that will probably be better...